source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnRenderer.java

```diff
@@ -104,6 +108,26 @@ else if ((column = Column.dynamic_cast(baseCardElement)) == null)
             returnedView.setBackgroundColor(Color.parseColor(color));
         }
 
+        BackgroundImage backgroundImageProperties = column.GetBackgroundImage();
+        if (backgroundImageProperties != null && !backgroundImageProperties.GetUrl().isEmpty())
+        {
+            BackgroundImageLoaderAsync loaderAsync = new BackgroundImageLoaderAsync(renderedCard, context, returnedView, hostConfig.GetImageBaseUrl(), backgroundImageProperties);
+
+            IOnlineImageLoader onlineImageLoader = CardRendererRegistration.getInstance().getOnlineImageLoader();
+            if(onlineImageLoader != null)
```

Comment:
nitpick: inconsistent spacing between `if` and `(`

Reply:
also in ContainerRenderer

---
In reply to: [256620650](https://github.com/Microsoft/AdaptiveCards/pull/2361#discussion_r256620650) [](ancestors = 256620650)